packages/services/f-metadata/README.md

```diff
@@ -89,43 +95,42 @@ brazePromise.then(brazeDispatcher => {
 
 ### Config Object
 
-
-#### `config.apiKey`
+`config.apiKey`
```

Comment:
It may be good to keep these as titles, as then Github automatically adds skip links to them (which is super useful when copy/pasting links to people asking about the various things in a component/service)